packages/react-core/src/components/Alert/AlertBody.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/Alert/styles.css';
+
+const propTypes = {
+  title: PropTypes.string,
+  children: PropTypes.node,
+  className: PropTypes.string
+};
+
+const defaultProps = {
+  title: null,
+  children: '',
+  className: ''
+};
+
+const RenderTitle = title => (
```

Comment:
not needed, but in the future, all functions should be `lowerCamelCased`.  Only React components should be `UpperCamelCased`. Since this one does not take a props object and is not instantiated by React it would need to be `lowerCamelCased`.

Another option is to make an `AlertTitle` component since you have already separated out all the other components.